libs/shared-shadcn-ui-components/src/components/ui/pagination.tsx

```diff
@@ -0,0 +1,144 @@
+import { Icon } from '@talent-connect/shared-atomic-design-components'
+import * as React from 'react'
+
+import {
+  ButtonProps,
+  buttonVariants,
+} from '@talent-connect/shared-shadcn-ui-components'
+import { cn } from '@talent-connect/shared-utils'
```

Comment:
We just discussed the naming of this and it still took me a minute to recall what `cn` means. Is it a big deal to rename it to something like `classnames`? 🤔

Reply:
We discussed it [here](https://github.com/talent-connect/connect/pull/953/files#r1724705754). 

If you want, we can rename it. But then we need to remember to manually update the import statement and the function name in each generated (or copy-pasted) shadcn/ui component where this function is used.

Reply:
Sounds good to me.

Reply:
I'm fine with simply `classNames`. There's a [very famous](https://github.com/JedWatson/classnames) library that does exactly that, used by 4.5 million repositories, so there's good precedence it works.